compiler/next/include/chpl/queries/Context.h

```diff
@@ -313,20 +313,11 @@ class Context {
    */
   void markUniqueCString(const char* s);
 
-  /**
-    Return the name of the module containing this ID.
-   */
-  UniqueString moduleNameForID(ID id);
```

Comment:
We're removing it because we don't use it but actually because `M.Inner` could be a module M containing a function `Inner` or a module M containing a module `Inner` it's not that easy. But we could pull out the top-level module name from the ID if we wanted to; I just don't know why we would need to. Additionally I've never really liked that Context.h talked about Modules because it's supposed to be query framework support (meaning I'd rather factor out Chapel-specific stuff like Modules elsewhere, like in `uAST` classes, if that's possible).